data/fairdatahub/RD3.yaml

```diff
@@ -0,0 +1,13 @@
+---
+name: RD3
+description: The Rare Disease Data about Data (RD3) sample tracker for Solve-RD
```

Comment:
A **very** minor edit. The "official" title is `Rare Disease Data about Data (RD3) database for Solve-RD`.

Reply:
Fixed! thanks for the suggestion :)